### DISCUSSION TITLE
DNS 8.8.8.8 set in client

### DISCUSSION THREAD

**Discussion by ridderr** [schwabe/ics-openvpn#1291] [Q&A] [answered]
I'm getting an DNS server set (8.8.8.8) in my client which is not defined in my client config. I cannot check the server because that's a closed TP-Link router but I haven't confugred that in my router.
I would like to have my VPN android client to use only the private DNS (pi-hole). How can I remove/ignore the 8.8.8.8 DNS in the VPN client.

**Comment by schwabe** [comment]
please attach a logfile

**Comment by ridderr** [comment]
See attached, I took some time to have a look at it and it looks like to come from the router. Would then be nice if it can be ignored.

[vpn-log.txt](https://github.com/schwabe/ics-openvpn/files/5836574/vpn-log.txt)

**Comment by schwabe** [comment, marked as answer]
Look at pull-filter in openvpn man page and then set that as custom options.  Something like `pull-filter ignore "dhcp-option DNS"`

**Reply by ridderr** [reply]
Super that's the solution .
